package.json

```diff
@@ -76,7 +76,7 @@
   },
   "devDependencies": {
     "@babel/core": "7.14.6",
-    "@jest/core": "27.3.1",
+    "@jest/core": "29.7.0",
```

Comment:
Jmping up two major versions.  Any concerns?

Reply:
No, there are bug fixes and some improvements but no breaking changes.